uta_tools/uta_tools.py

```diff
@@ -42,3 +45,392 @@ def __init__(self, seqrepo_data_path: str = SEQREPO_DATA_PATH,
         self.mane_transcript = MANETranscript(
             self.seqrepo_access, self.transcript_mappings,
             self.mane_transcript_mappings, self.uta_db)
+
+    async def transcript_to_genomic(
+            self, transcript: str, exon_start: int, exon_end: int,
+            exon_start_offset: int = 0, exon_end_offset: int = 0,
+            gene: str = None, *args, **kwargs) -> Optional[GenomicData]:
+        """Get genomic data given transcript data.
+        Will liftover to GRCh38 coordinates if possible.
+
+        :param str transcript: Transcript accession
+        :param int exon_start: Starting transcript exon number
+        :param int exon_end: Ending transcript exon number
+        :param int exon_start_offset: Starting exon offset
+        :param int exon_end_offset: Ending exon offset
+        :param str gene: Gene symbol
```

Comment:
it looks like some of these (gene and transcript) should be hinted as `Optional`